.github/workflows/eks.yaml

```diff
@@ -2,12 +2,12 @@ name: EKS (ENI)
 
 # Any change in triggers needs to be reflected in the concurrency group.
 on:
-  pull_request: {}
+  pull_request_target: {}
   schedule:
     - cron:  '10 */6 * * *'
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.event.after || 'scheduled' }}
+  group: ${{ github.workflow }}-${{ github.event.pull_request_target.number || github.event.after || 'scheduled' }}
```

Comment:
I don't think there is such a thing as `github.event.pull_request_target`. All jobs are currently cancelling each other because of this. I'll send a fix.